src/lib/destinations.ts

```diff
@@ -75,6 +76,23 @@ export const deliverToDestination = async (
       await bucketsClient.send(new PutObjectCommand(putCommandArgs));
 
       break;
+
+    case "sftp":
```

Comment:
note: as we add more destination types, I'll likely break the actual delivery logic for each type out into separate files in a future PR

Reply:
agreed, I think we're there.